Added verify command

htgar opened the pull request:
Added &verify command to dm all users with the @unverified tag (no arguments taken). This command can be called by the @OIC, @school-leads or @temp_admin.

htgar committed:
Added verify command

Added &verify command to dm all users with the @unverified tag (no arguments taken). This command can be called by the @OIC, @school-leads or @temp_admin.

joelleoqiyi commented on index.js:
```diff
@@ -376,5 +378,27 @@ const testCommand = (arguments, receivedMessage) => {
+    );
+  }
+  else{
+    if (receivedMessage.member.roles.cache.some(r => r.name === "OIC" || r.name === "school-leads" || r.name === "temp_admin")){
```
@Iscaraca would it be better if instead of using `r.name` we check for the roleID?

htgar committed:
Added verify command

verify command to DM all users with the tag @unverified. Can only be performed by @OIC, @shcool-leads and @temp_admin

htgar committed:
Delete package-lock.json

joelleoqiyi commented on index.js:
```diff
@@ -376,5 +379,25 @@ const testCommand = (arguments, receivedMessage) => {
+  else{
+    if (receivedMessage.member.roles.cache.has("698842106613202944") || receivedMessage.member.roles.cache.has("698829800965668884") || receivedMessage.member.roles.cache.has("699065951123013652")){
+      receivedMessage.guild.roles.cache.get("698841628856811601").members.forEach(m => {
+          return m.send("Please verify yourself. For help, dm the OIC or any of the school leads");
```
Can you use the same message that is used in the original confirm message here instead? ie how they shud verify. 

Since most of the people would not be able to find the original message.

htgar committed:
Changed message

ThinkerPal commented:
Looks all good, going to merge.
